README.md

```diff
@@ -1,3 +1,5 @@
+Trello link here: https://trello.com/b/ffCjlMki/js-tuesday
+
 REBASE ME!
 
 All your re-Bass are belong to us
```

Comment:
Interesting. Have you considered using a lower-case 'b'? I find that it makes your sentence clearer. [For an example of lowercases, see a book]